### ISSUE TITLE
New line inside code blocks

### ISSUE THREAD

**Issue by tpoxa** [quasarframework/quasar-ui-qmarkdown#128]
**Describe the bug**
\n Newline symbols should be ignored inside a code block variables

**To Reproduce**
Go to https://quasarframework.github.io/quasar-ui-qmarkdown/demo
Put next code inside
<pre>
```js
hello("Content-type: application/x-www-form-urlencoded\r\n")
```
</pre>

**Expected behavior**
I would like to see sth like 

hello("Content-type: application/x-www-form-urlencoded\r\n")

**Actual behavior

hello("Content-type: application/x-www-form-urlencoded\r
")

Maybe its Prism related issue?

**Comment by hawkeye64** [comment]
@tpoxa Add `no-breaks` property

**Comment by tpoxa** [comment]
Thank you for your reply. It does not work, even on demo page.

**Comment by tpoxa** [comment]
@hawkeye64 Is `no-breaks` the same as Disable breaks on demo page?

**Comment by hawkeye64** [comment]
@tpoxa From markdown-it site:
```
// full options list (defaults)
var md = require('markdown-it')({
  html:         false,        // Enable HTML tags in source
  xhtmlOut:     false,        // Use '/' to close single tags (<br />).
                              // This is only for full CommonMark compatibility.
  breaks:       false,        // Convert '\n' in paragraphs into <br>
  langPrefix:   'language-',  // CSS language prefix for fenced blocks. Can be
                              // useful for external highlighters.
  linkify:      false,        // Autoconvert URL-like text to links

  // Enable some language-neutral replacement + quotes beautification
  typographer:  false,

  // Double + single quotes replacement pairs, when typographer enabled,
  // and smartquotes on. Could be either a String or an Array.
  //
  // For example, you can use '«»„“' for Russian, '„“‚‘' for German,
  // and ['«\xA0', '\xA0»', '‹\xA0', '\xA0›'] for French (including nbsp).
  quotes: '“”‘’',

  // Highlighter function. Should return escaped HTML,
  // or '' if the source string is not changed and should be escaped externaly.
  // If result starts with <pre... internal wrapper is skipped.
  highlight: function (/*str, lang*/) { return ''; }
});
```
Notice the `breaks` option?
This is how QMarkdown handles it:
```js
      const opts = {
        html: this.__isEnabled(this.noHtml),
        linkify: this.__isEnabled(this.noLinkify),
        typographer: this.__isEnabled(this.noTypographer),
        breaks: this.__isEnabled(this.noBreaks),
        highlight: highlight
      }
```

**Comment by tpoxa** [comment]
I thought a good idea is to try this option online. So I went to the demo page https://quasarframework.github.io/quasar-ui-qmarkdown/demo
There is an option. Disable breaks. I enabled it. Pasted my markdown code and I am still seeing the new line. What am I doing wrong? Thank you for your patience.

**Comment by hawkeye64** [comment]
@tpoxa I'll have to look into the demo page. Seems other things are broken/not connected properly (ie: highlighting still is one when disabled)

**Comment by hawkeye64** [comment]
@hawkeye64 Sorry it took a while. I have figured out the issue.
If you use the interactive editor to paste in some multi-text, without any other markup applied, "disable breaks". You will see it in action.
Now, put that same text in a code block and try "disabling breaks"
Does not work. That's because code blocks are inside `<pre><code> <!-- here --></code></pre>`
When this happens, these HTML tags interpret everything in them. They don't need `\n` converted to `<br>` as they do this on their own.
So, basically, if you don't want breaks, then don't put it in a code block. Since this is one line, I would bold or mark it.
